ccl/importccl: TestImportIntoCSV failed

[(ccl/importccl).TestImportIntoCSV failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2833372&tab=buildLog) on [master@fe14895b103aae7f969ebf90763802629ad209c5](https://github.com/cockroachdb/cockroach/commits/fe14895b103aae7f969ebf90763802629ad209c5):

```
=== RUN   TestImportIntoCSV
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportIntoCSV845252864
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportIntoCSV
    import_stmt_test.go:3433: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportIntoCSV845252864
--- FAIL: TestImportIntoCSV (78.12s)
=== RUN   TestImportIntoCSV/empty-file
    import_stmt_test.go:2879: error scanning '&{0xc00207c120 <nil>}': pq: relation "t" is offline: importing
    --- FAIL: TestImportIntoCSV/empty-file (1.46s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestImportIntoCSV PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestImportIntoCSV.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/importccl).TestImportIntoCSV failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2833759&tab=buildLog) on [master@c6125c3c5f4e416382c19adfaebe6c2190977190](https://github.com/cockroachdb/cockroach/commits/c6125c3c5f4e416382c19adfaebe6c2190977190):

```
=== RUN   TestImportIntoCSV
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportIntoCSV267930830
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportIntoCSV
    import_stmt_test.go:3433: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportIntoCSV267930830
--- FAIL: TestImportIntoCSV (68.54s)
=== RUN   TestImportIntoCSV/empty-with-files
    import_stmt_test.go:2891: error scanning '&{0xc00ecd9c20 <nil>}': pq: relation "t" is offline: importing
    --- FAIL: TestImportIntoCSV/empty-with-files (0.70s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestImportIntoCSV PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestImportIntoCSV.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/ccl/importccl/import_stmt_test.go
+++ b/pkg/ccl/importccl/import_stmt_test.go
@@ -2611,7 +2611,6 @@ func TestExportImportRoundTrip(t *testing.T) {
 // -> Rollback of a failed IMPORT INTO
 func TestImportIntoCSV(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	skip.WithIssue(t, 62853, "flaky test")
 	defer log.Scope(t).Close(t)
 
 	skip.UnderShort(t)

